docs/math-support.md

```diff
@@ -0,0 +1,27 @@
+# Math Support
+
+The published pages don't support math formulas by default. To enable this feature, you can add the following code snippet to the end of `_layouts/page.html`:
```

Comment:
I like the trick, would it be helpful to provide an example on how to write a math formula in markdown? Possibly with a link to the reference syntax?

Reply:
@TaiChi-IO agreed with @riccardoferretti, could you provide an example of the syntax you would use?

Reply:
yes, please see commit https://github.com/foambubble/foam/pull/109/commits/525b8cf4039df841b068b48bfee3910753e86f07